extensions/classify/config.proto

```diff
@@ -0,0 +1,98 @@
+/* Copyright 2020 Istio Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+
+package istioclassify;
+
+// Configuration to classify traffic.
+message PluginConfig {
+  // next id: 2
+  // Multiple independent classification configurations.
+  repeated Classification classifications = 1;
+}
+
+// Classification uses [well known attributes](https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/security/rbac_filter#condition) as inputs and produces
+// a new attribute that denotes a specific classification.
+// The new attribute can be used by a downstream plugin.
```

Comment:
plugin vs filter which is preferable?

Reply:
We call it plugin in the Wasm paralance, so I prefer plugin.